packages/services/f-stat-client/src/statClient.js

```diff
@@ -0,0 +1,55 @@
+const { Client } = require('@elastic/elasticsearch');
+
+export default class StatClient {
+    constructor (
+        url,
+        port,
+        tenant,
+        featureName,
+        user = null,
+        pwd = null,
+        indexName = 'justeat',
+        mock = null
+    ) {
+        const conn = {
+            node: `${url || 'http://localhost'}:${port || 9200}`
+        };
+
+        if (user) {
+            conn.auth = {
+                username: user,
+                password: pwd
+            };
+        }
+
+        if (mock) {
+            conn.Connection = mock.getConnection();
+        }
+
+        this.client = new Client(conn);
+        this.tenant = tenant || 'ns';
+        this.feature = featureName || 'NotSpecified';
+        this.indexName = indexName;
+    }
+
+    async publish ({
+        verb, segment, status, timing
+    }) {
+        const payload = {
+            index: this.indexName,
+            body: {
```

Comment:
I think this can be a combination of fixed params and dynamic ones. For ex

```
body: {
  tenant: this.tenant
  feature: this.feature,
  ...payload
}
```